modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/UndoRedo.js

```diff
@@ -481,15 +482,22 @@ export default function UndoRedo({
 						className="ml-2"
 						onActiveChange={setActive}
 						trigger={
-							<ClayButtonWithIcon
+							<ClayButton
 								aria-label={Liferay.Language.get('history')}
```

Comment:
Tested and changes LGTM. 

The only thing I would to is to reduce a bit the left-right padding to this button so it looks better, by adding class `.px-2`, but this
<img width="703" alt="Screenshot 2024-10-09 at 12 00 36" src="https://github.com/user-attachments/assets/416dce92-6c35-4af6-ad46-5027687c5cb4">

And currently it is: 
<img width="703" alt="Screenshot 2024-10-09 at 12 02 42" src="https://github.com/user-attachments/assets/34aa51f9-cfd8-4e08-b7de-bd780dc2f005">

 is optional. This way it would look like: 
<img width="703" alt="Screenshot 2024-10-09 at 12 02 42" src="https://github.com/user-attachments/assets/8deb14c6-8461-45e6-b654-9ec88002ac37">

Reply:
Hey @ambrinchaudhary ,

Thanks for pointing that out, it looks much better I make that change and push it